general/base_import/import_faq.rst

```diff
@@ -0,0 +1,201 @@
+As an example, here is purchase.order_functional_error_line_cant_adpat.CSV file of some quotations you can import, based on demo data.
+
+`File for some Quotations <../../_static/example_files/purchase.order_functional_error_line_cant_adpat.csv>`_.
+
+The following CSV file shows how to import purchase orders with their respective purchase order lines:
+
+`Purchase orders with their respective purchase order lines <../../_static/example_files/o2m_purchase_order_lines.csv>`_.
+
+The following CSV file shows how to import customers and their respective contacts:
+
+`Customers and their respective contacts <../../_static/example_files/o2m_customers_contacts.csv>`_.
+
+
+
+Can I import several times the same record?
+-------------------------------------------
+
+If you import a file that contains one of the column "External ID" or "Database ID", records that have already been imported will be modified instead of being created. This is very usefull as it allows you to import several times the same CSV file while having made some changes in between two imports. Odoo will take care of creating or modifying each record depending if it's new or not.
+
+This feature allows you to use the Import/Export tool of Odoo to modify a batch of records in your favorite spreadsheet application.
+
+
+
+What happens if I do not provide a value for a specific field?
+--------------------------------------------------------------
+
+If you do not set all fields in your CSV file, Odoo will assign the default value for every non defined fields. But if you set fields with empty values in your CSV file, Odoo will set the EMPTY value in the field, instead of assigning the default value.
+
+
+
+How to export/import different tables from an SQL application to Odoo?
+----------------------------------------------------------------------
+
+If you need to import data from different tables, you will have to recreate relations between records belonging to different tables. (e.g. if you import companies and persons, you will have to recreate the link between each person and the company they work for).
+
+To manage relations between tables, you can use the "External ID" facilities of Odoo. The "External ID" of a record is the unique identifier of this record in another application. This "External ID" must be unique accoss all the records of all objects, so it's a good practice to prefix this "External ID" with the name of the application or table. (like 'company_1', 'person_1' instead of '1')
+
+As an example, suppose you have a SQL database with two tables you want to import: companies and persons. Each person belong to one company, so you will have to recreate the link between a person and the company he work for. (If you want to test this example, here is a <a href="/base_import/static/csv/database_import_test.sql">dump of such a PostgreSQL database</a>)
+
+We will first export all companies and their "External ID". In PSQL, write the following command:
```

Comment:
Put a link on the "In PSQL" to point to http://www.postgresql.org/docs/9.5/static/app-psql.html